test/pottery/scan_test.clj

```diff
@@ -56,3 +56,9 @@
                                (defn square [x] x)
                                (defn render-thing [{:keys [some-arg]}]
                                  [:div (tr "Some text %1" some-arg)]))}))))
+
+(deftest scan-files-test
+  (is (= '(#:pottery.scan{:filename "test-resources/foo.cljc",
+                          :expressions (#:pottery.scan{:value "Some text %1 %2 %3"})})
```

Comment:
In terms of style we prefer to avoid namespaced maps, so I'd prefer these 3 keywords to be aligned with the other occurences in the namespace.

Reply:
Yeah I'm fine with that, it's just the way how CIDER printed it